src/koans/26_transducers.clj

```diff
@@ -0,0 +1,23 @@
+(ns koans.26-transducers
+  (:require [koan-engine.core :refer :all]))
+
+(def xfms
+  (comp (map inc)
+  (filter even?)))
+
+(meditations
+ "Consider that sequence operations can be used as transducers"
+ (= __
+    (transduce xfms conj [1 2 3]))
```

Comment:
This feels like it could kind of a large first step - I wonder if there's a smaller initial example to put before this one? Specifically, maybe a transformation that doesn't have a `comp` in it? I think then this  one and the rest of them flow really well.

Reply:
I'll have a think and try and come up with something...